use packaged libxmp instead of bundled libxmp-lite, works around
build issue when libxmp package is installed

Index: source/audiolib/src/xmp.cpp
--- source/audiolib/src/xmp.cpp.orig
+++ source/audiolib/src/xmp.cpp
@@ -9,7 +9,7 @@
 #include "pragmas.h"
 
 #define BUILDING_STATIC
-#include "libxmp-lite/xmp.h"
+#include <xmp.h>
 
 int  MV_GetXMPPosition(VoiceNode *voice)               { return voice->position; }
 void MV_SetXMPPosition(VoiceNode *voice, int position) { xmp_seek_time((xmp_context)voice->rawdataptr, position); }
